capabilitytypes/tosca.capabilities/Compute/CapabilityType.tosca

```diff
@@ -36,3 +36,6 @@ capability_types:
         status: supported
         constraints:
           - greater_or_equal: 0 MB
+    metadata:
```

Comment:
Has this been produced by Winery? (line 6 also has metadata)

Reply:
> Has this been produced by Winery? (line 6 also has metadata)

Yes, all files have been produced by Winery. At was the latest version at the time of the pull request. Do I need to change something?

Reply:
This is weird because the `metadata` block has been added twice. Never seen this before. You are sure that you haven't added this manually?

Reply:
Could it be something related to an automatic Git merge? For sure, this particular type is not something we are concerned with from the CTT perspective. Shall I just remove lines 39-41?